webapp/src/simulator.ts

```diff
@@ -45,6 +45,11 @@ export function init(root: HTMLElement, cfg: SimulatorConfig) {
     root.appendChild(debuggerDiv);
 
     const nestedEditorSim = /nestededitorsim=1/i.test(window.location.href);
+    let parentOrigin: string | null = null;
```

Comment:
i don't think we use | null checking in this part of pxt yet.

Reply:
Is this due to some limitation that I am currently unaware of, a style decision, or something else? Should I use `| undefined` instead? We currently use let variable: type | null in the `DebuggerVariablesState` interface, and in `monacoEditAmendments` in pxt/webapp. There are more instances of type | undefined used in pxt/webapp.

Reply:
I poked Daryl and he was able to clarify. Appending `| null` to a type is only necessary when strict null checking is enabled. Otherwise, all types are implicitly unioned with `null` and `undefined`. Only the pxt/pxtpy enabled strict null checking. So I'll remove `| null` from the type.

Reply:
I've removed the `| null` from the variable declaration.